examples/tools.html

```diff
@@ -0,0 +1,12 @@
+<script type="module">
+    import {HanskenClient} from './src/hansken-js.js';
+    const hansken = new HanskenClient('/gatekeeper', '/keystore');
+
+    hansken.scheduler().tools()
+        .then(tools => Object.keys(tools).forEach(tool => {
+            document.write(tool + ': ' + tools[tool].description + '<br>')
+        }))
+        .then(() => hansken.scheduler().toolsBuilder())
+        .then(builder => builder.enable('HelloWorldPlugin').disable('imaging/picture').build())
```

Comment:
When preparing a list of tools for an extraction with hansken.py, I usually just use a `set` with the default enabled tools, and start adding and removing from there. If this builder pattern is more in line with what the rest of `hansken-js` uses, it's okay with me though.

Reply:
yeah, but js doesn't have sets, only arrays. This felt safer.

Reply:
Also, In my case it validates tool names. We discussed it here (@nfi-jd  and @maikelBoom ) and we stick with the current toolBuilder